schedule.md

```diff
@@ -1,3 +1,13 @@
 | Unit |  Date      | Lesson |
 |:----:|:----------:|--------|
 | 0    | 2016-07-18 | [Environment & Tools](lessons/env-and-tools) |
+| 0    | 2016-07-18 | [Intro to Pair Programming]() |
+| 1    | 2016-07-19 | [JS Fundamentals: Workshop Review](lessons/javascript-fundamentals/workshop-review) |
+| 1    | 2016-07-20 | [JS Fundamentals: Types, Strings, and Numbers](lessons/javascript-fundamentals/types-strings-and-numbers) |
+| 1    | 2016-07-21 | [JS Fundamentals: Objects and Arrays](lessons/javascript-fundamentals/objects-and-arrays) |
+| 1    | 2016-07-22 | [Intro To HTML and CSS Part 1](lessons/html-and-css/intro-to-html-and-css-part-1) |
+| 1    | 2016-07-25 | [JS Fundamentals: Functions Deep Dive](lessons/javascript-fundamentals/functions-deep-dive) |
+| 1    | 2016-07-26 | [JS Fundamentals: Advanced Array Methods](lessons/javascript-fundamentals/advanced-array-methods) |
+| 1    | 2016-07-27 | [Intro To HTML and CSS Part 2](lessons/html-and-css/intro-to-html-and-css-part-2) |
+| 1    | 2016-07-28 | [Review/Project]() |
+| 1    | 2016-07-29 | [Asessment]() |
```

Comment:
1. Yeah good point...I need to go through and add some learning standards. I'm still a little unclear about the difference between standards and objectives...What are some examples of standards we'd be working towards over these first two weeks?

2. Yeah demo would be mostly live coding by me, but definitely would like to mix in some other teaching methods as well. I like the ideas of getting students involved by having them research on their own and/or prepare short demos. I'll also start planning out more challenging discussion questions